app/lib/client_sorter.rb

```diff
@@ -96,7 +96,11 @@ def filtered_clients
     end
 
     if @filters[:vita_partners].present?
-      ids = JSON.parse(@filters[:vita_partners]).map { |vita_partner| vita_partner["id"] }
+      # For backwards compatibility for users in the middle of a search while this is deployed,
```

Comment:
do you want to make another ticket, or a subtask of this one, to remove the hash-handling code a few days after this is pushed?

Reply:
I think it's okay to leave it as a comment in the code for now with the capability of handling both. We can remove it at our leisure.